libs/platform/auth/login/login.component.ts

```diff
@@ -75,7 +84,12 @@ export class AuthLoginComponent extends ContentMixin<LoginOptions>(LitElement) {
   }
 
   protected override render(): TemplateResult {
-    return html` ${when(
+    return html`
+      <oryx-heading as=${HeadingTag.H2}>
+        <h1>${this.i18n('user.login.log-in')}</h1>
+      </oryx-heading>
```

Comment:
We could add the text outside the login component inside a text component, to avoid hard coding it.